src/user-data-converter.ts

```diff
@@ -23,9 +20,8 @@ const createCourseCreditRange = (course: CornellCourseRosterCourse): readonly [n
 };
 
 export const cornellCourseRosterCourseToAppCourse = (
```

Comment:
Should we rename this `cornellCourseRosterCourseToFirestoreSemesterCourse` to be consistent?

Reply:
renamed